ui/core/src/model/dashboard.ts

```diff
@@ -25,6 +26,7 @@ export interface DashboardResource {
 }
 
 export interface DashboardSpec {
+  display?: Display;
```

Comment:
Is this mostly for consistency? Curious if there is overlap with how `metadata` is used--for instance, when would display.name be used compared with metadata.name?

Reply:
Yeah it's for consistency and also because in `metadata.name` you cannot put space. So I was thinking maybe you would like to put some space in the dashboard title.
Basically if `disply.name` is used, it replaced metadata.name in the Breadcrumbs.

But I'm not super sure it's a great idea.

Reply:
Ah okay I see, sorry it took me a while to follow up here! And it's optional so this isn't a breaking change or anything right?

Reply:
yup no breaking change here. If it's not used, then `metadata.name` should be used :)